cb-talent-development/employer-style-base
# EM-1237: Why CareerBuilder Layout and Placeholders

**Purpose**
> As a potential buyer or high-level decision maker, I want to visit Hiring Dot and know what CareerBuilder can offer me as a vendor/partner.

* Adds several sections for the Why CB page
* Refactors existing code into new mixins
** `container-edge-calc` (buy box)
** `item-bar__container` (homepage stats)
** `text-image-panels--full-bleed-image` (small biz cta)
** `quad-layout` (small biz benefits)
* Adds `header--extra-large` partial and styles
* I've started prefixing variables with underscore when they are specific to one thing, per https://github.com/airbnb/css#variables
* I've also decided not to create variables _until_ I need the value more than once, so there are lots of un-variabled values

**JIRA**
https://cb-content-enablement.atlassian.net/browse/EM-1237

**Changes**
* Improvements and fixes, Changes to developer setup/environment,  Architectural changes, Migrations or Steps to Take on Production
  * n/a
  
* Library changes
  * bump style base

* Side effects
  * n/a

**Screenshots**
n/a

**Feature Server**
http://web.employer-1.development.c66.me/company/why-careerbuilder
http://cb-talent-development.github.io/employer-style-base/index.html

**How to Verify These Changes**
* Specific pages to visit
  * http://web.employer-1.development.c66.me/company/why-careerbuilder

* Steps to take
  * Compare the Why CB page to the AC in the Jira story
  * Visit the pages that were refactored above and verify that the components look the same

* Responsive considerations
  * View the Why CB page on various devices/viewports

**Relevant PRs/Dependencies**
  * 

**Additional Information**
